bftclient/test/bft_client_api_tests.cpp

```diff
@@ -38,6 +38,7 @@ using namespace bftEngine::impl;
 using namespace bftEngine;
 using namespace placeholders;
 using namespace concord::secretsmanager;
+using ReplicaId_t = bft::client::ReplicaId;
```

Comment:
Why not just `using bft::client::ReplicaId`?

Reply:
Possible, but this structure repeats 16 times in the file.
My idea is that it might make the code less readable.

Reply:
Just curious - what changed so that we need to refer ReplicaId with the whole namespace>